CMakeLists.txt

```diff
@@ -551,14 +551,24 @@ MESSAGE(
 "* GIG player                  : ${STATUS_GIG}\n"
 )
 
+#MESSAGE(
+#"\n"
+#"-----------------------------------------------------------------\n"
+#"IMPORTANT:\n"
+#"after installing missing packages, remove CMakeCache.txt before\n"
+#"running cmake again!\n"
+#"-----------------------------------------------------------------\n"
+#"\n\n")
+
+IF(LMMS_HOST_X86_64 AND LMMS_SUPPORT_VST)
 MESSAGE(
 "\n"
 "-----------------------------------------------------------------\n"
 "IMPORTANT:\n"
-"after installing missing packages, remove CMakeCache.txt before\n"
-"running cmake again!\n"
+"For VST we need 32bit QtCore libraries installed but cmake will not check for 32bit version.\n"
```

Comment:
Via [this thread](http://marc.info/?l=cmake&m=129865216611753), we may need to temporarily switch out the patch to `qmake`, do a `find_package` and then switch back.  If someone can find a command line switch for locating the 32-bit libs, I can help write up a CMake command to execute and return the value as well.